clevertap-core/src/main/java/com/clevertap/android/sdk/inbox/CTInboxListViewFragment.java

```diff
@@ -42,8 +44,8 @@ public class CTInboxListViewFragment extends Fragment {
 
     interface InboxListener {
 
-        void messageDidClick(Context baseContext, CTInboxMessage inboxMessage, Bundle data,
-                HashMap<String, String> keyValue,boolean isBodyClick);
+        void messageDidClick(Context baseContext, int position, CTInboxMessage inboxMessage, Bundle data,
```

Comment:
give callback after firing deep link and push primer dialog

Reply:
this is done.